spec.bs

```diff
@@ -7284,6 +7338,31 @@ for a given [=k-anonymity key=]. These records are stored in the [=user agent=].
   :: The [=moment=] when the k-anonymity status in this record was last fetched.
 </dl>
 
+<h3 id=auction-data-config-header>auction data config</h3>
+
+An <dfn>auction data config</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="auction data config">
+  :  <dfn>seller</dfn>
+  :: An [=origin=].
+    The origin of the seller running the ad auction. The [=origin/scheme=] must be "`https`".
+  :  <dfn>coordinator</dfn>
+  :: The origin of the coordinator hosting public encryption keys for the server
+    running the ad auction. The [=origin/scheme=] must be "`https`".
+  :  <dfn>request size</dfn>

```

Comment:
just to double check, so will these be used by the IETF spec, and this W3C spec does not need to worry about using it?

Reply:
The IETF spec doesn't need to know this as it just takes the key, but the method the browser uses to get the keys from a coordinator and seller is explicitly left unspecified.